devices/ITCommander.js

```diff
@@ -3,12 +3,40 @@ const exposes = require('../lib/exposes');
 const e = exposes.presets;
 const reporting = require('../lib/reporting');
 
+const bind = async (endpoint, target, clusters) => {
+    for (const cluster of clusters) {
+        await endpoint.bind(cluster, target);
+    }
+};
+
+const fzclick = {
+    diyruz_freepad_clicks: {
```

Comment:
Please move this converter to fromZigbee.js

Reply:
thanks , moved